doc/Protocols/pr-meta_x-road_service_metadata_protocol.md

```diff
@@ -92,28 +108,11 @@ In addition, it is possible to retrieve a list of clients in other, federated X-
 
 Thus, in order to retrieve a list of clients defined in the X-Road instance `AA`, the request URL is `http://SECURITYSERVER/listClients?xRoadInstance=AA`.
 
-Security server MUST respond with content-type `text/xml` and the response MUST contain the `clientList` XML element defined below
-(full XML schema appears in Annex [A](#annex-a-xml-schema-for-messages)).
-Annex [C.1](#c1-listclients-response) contains an example response message.
+It is possible to control the response content type using HTTP `Accept` header. If the header value is `application/json`, the security server must produce an application/json response, as defined in Annex B, [OpenAPI definition](#openapi-definition). Otherwise, security server MUST respond with content-type `text/xml` and the response MUST contain the `clientList` XML element defined in Annex [A](#annex-a-xml-schema-for-messages)).
 
-```xml
-    <xs:element name="clientList" type="ClientListType"/>
-    <xs:complexType name="ClientListType">
-        <xs:sequence>
-            <xs:element maxOccurs="unbounded" minOccurs="0"
-                name="member" type="ClientType"/>
-        </xs:sequence>
-    </xs:complexType>
-    <xs:complexType name="ClientType">
-        <xs:sequence>
-            <xs:element name="id" type="id:XRoadClientIdentifierType"/>
-            <xs:element name="name" type="xs:string" minOccurs="0"/>
-        </xs:sequence>
-    </xs:complexType>
-```
+Annex [C.1](#c1-listclients-response) contains an example XML and JSON response messages
 
-The `XRoadClientIdentifierType` represents a globally unique identifier of an X-Road client.
-The client identifier has a hierarchical structure consisting of X-Road instance, member class, member and (optionally) subsystem codes. See specification \[[PR-MESS](#Ref_PR-MESS)\] for explanation and specification of identifiers.
+The X-Road client identifier has a hierarchical structure consisting of X-Road instance, member class, member and (optionally) subsystem codes. See specification \[[PR-MESS](#Ref_PR-MESS)\] for explanation and specification of identifiers.
 
 ## 3 Retrieving List of Central Services
```

Comment:
Since `listCentralServices` seems to be implemented also in this pull request, the documentation should be updated also (i.e. can return also JSON)

Reply:
Removed the incomplete implementation (addressed in a separate issue).